src/utils/constants.ts

```diff
@@ -0,0 +1,2 @@
+const API_PATH = 'http://localhost:5000/api';
+export default API_PATH;
```

Comment:
Remove both lines and replace with:

`export const API_PATH = 'http://localhost:5000/api';`